source/_components/camera.xiaomi.markdown

```diff
@@ -53,18 +53,42 @@ camera:
     name: Camera
     host: '192.168.1.100'
     model: 'yi'
-    password: my_password_123
+    password: YOUR_PASSWORD
 ```
 
-Configuration variables:
-
-- **name** (*Required*): A human-friendly name for the camera.
-- **host** (*Required*): The IP address or hostname of the camera.
-- **model** (*Required*): The model of Xiaomi Camera, currently supporting yi and xiaofang.
-- **password** (*Required*): The password to the FTP server on the camera (from above), can be any string as the current firmware doesn't allow setting ftp passwords.
-- **path** (*Optional*): The path to the raw MP4 files. Defaults to `/media/mmcblk0p1/record`.
-- **username** (*Optional*): The user that can access the FTP server. Defaults to `root`.
-- **ffmpeg_arguments** (*Optional*): Extra options to pass to `ffmpeg` (e.g., image quality or video filter options).
+{% configuration %}
+name:
+  description: A human-friendly name for the camera.
+  required: true
+  type: string
+host:
+  description: The IP address or hostname of the camera.
+  required: true
+  type: string
+model:
+  description: The model of Xiaomi Camera, currently supporting yi and xiaofang.
+  required: true
+  type: string
+password:
+  description:The password to the FTP server on the camera (from above), can be any string as the current firmware doesn't allow setting ftp passwords.
```

Comment:
Space after `:` is needed.

Not sure if it will fix the build issue though. It is pointing to a different place